app-creator/client/src/utils/helpers.ts

```diff
@@ -46,3 +46,12 @@ export const chips = [
     device: DeviceType.mobile,
   },
 ];
+
+/*
+ * Generates random ids with length 32.
+ */
+export function generateRandomId() {
+  return [...Array(32)]
+    .map((_) => (~~(Math.random() * 36)).toString(36))
```

Comment:
Can this be replaced with `Array.fill()`?

Reply:
Array.fill will calculate this value `(~~(Math.random() * 36)).toString(36))` one time and populate all elements in the array with it (i.e. ["a", "a", "a", ...]).